backend/src/test/controller/PromotionController.test.ts

```diff
@@ -203,10 +242,12 @@ describe('Unit tests for PromotionController', function () {
       .end((err, res) => {
         const frontEndErrorObject = res.body;
         expect(frontEndErrorObject?.errorCode).toEqual('ValidationError');
-        expect(frontEndErrorObject.message).toHaveLength(1);
+        expect(frontEndErrorObject.message).toHaveLength(3);
         expect(frontEndErrorObject.message[0]).toContain(
           '"cuisine" must be one of'
         );
+        expect(frontEndErrorObject.message[1]).toContain('"lat" is required');
+        expect(frontEndErrorObject.message[2]).toContain('"lon" is required');
```

Comment:
I don't think this is needed since you created a test `'POST /promotions/ - should not be able to add promotion if lat/lon do not exist'` that seems to test the same thing